sonnenbatterie/const.py

```diff
@@ -3,9 +3,8 @@
 
 
 BATTERY_LOGIN_TIMEOUT=120
```

Comment:
It may be useful to call the timeouts DEFAULT...
BATTER_LOGIN_TIMEOUT_DEFAULT to make clear that they are overrideable. 
What do you think?

Reply:
changed var names to begin with DEFAULT, did some tidying up of the time of use test case and pushed the update. Looks like having multiple authors is not a good idea so added a new contributors.txt file